<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFriends</class>
 <widget class="QDialog" name="AddFriends">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(241, 241, 241)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame{
border-radius:10px
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="DialogTitleBar" name="titleBar" native="true">
        <property name="styleSheet">
         <string notr="true">
#titleBar QPushButton{
	border-top-right-radius:10px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="nameInputEdit">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
        border: 1px solid #ccc;
        border-radius: 10px;
        padding: 6px 10px;
        font-size: 14px;
        background-color: #f9f9f9;
    }
    QLineEdit:focus {
        border: 1px solid #0078d7;
        background-color: #ffffff;
    }</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSelect">
           <property name="styleSheet">
            <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: white;
    background-color: rgb(53, 93, 144);
	color: rgb(255, 255, 255);
    border-radius: 8px;
    border-style: none;
    border-width: 1px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: rgb(93, 125, 166);
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
           </property>
           <property name="text">
            <string>查 询</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QListView" name="strangerList">
           <property name="styleSheet">
            <string notr="true">QListView {
    border: none;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DialogTitleBar</class>
   <extends>QWidget</extends>
   <header>View/Components/dialogtitlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
